driver.go

```diff
@@ -23,66 +23,69 @@ import (
 const (
 	driverName = "triton"
 	flagPrefix = driverName + "-"
-	envPrefix  = "TRITON_"
+	// SDC_ is for historical reasons
+	envPrefix  = "SDC_"
 )
 
 var (
-	defaultSdcAccount = ""
-	defaultSdcKeyFile = os.Getenv("HOME") + "/.ssh/id_rsa"
-	defaultSdcKeyId   = ""
-	defaultSdcUrl     = "https://us-east-1.api.joyent.com"
+	defaultTritonAccount = ""
+	defaultTritonKeyPath = os.Getenv("HOME") + "/.ssh/id_rsa"
+	defaultTritonKeyId   = ""
+	defaultTritonUrl     = "https://us-east-1.api.joyent.com"
 
 	// https://docs.joyent.com/public-cloud/instances/virtual-machines/images/linux/debian#debian-8
-	defaultSdcImage   = "debian-8"
-	defaultSdcPackage = "g3-standard-0.25-kvm"
+	defaultTritonImage   = "debian-8"
+	defaultTritonPackage = "g3-standard-0.25-kvm"
+
+	errUnimplemented = errors.New("UNIMPLEMENTED")
```

Comment:
No, just noting that fixing the compilation error I'd introduced was something else I'd staged locally (already pushed the fix in https://github.com/joyent/docker-machine-driver-triton/commit/5456b1f15a4e26d195dad0b5df2019569a40457d).  Sorry for the confusion!